src/forms/states/generate-http-reducers.ts

```diff
@@ -0,0 +1,71 @@
+import * as conf from '../../conf';
+import * as path from 'path';
+import {indent, writeFile} from '../../utils';
+import {Config} from '../../generate';
+import {ControllerMethod, ResponseDef} from '../../requests/requests.models';
+import {ProcessDefinition} from '../../definitions';
+
+export function GenerateHttpReducers(config: Config, name: string, dashedName: string, simpleName: string,
+                                     responseDef: ResponseDef, methods: ControllerMethod[],
+                                     schemaObjectDefinitions: ProcessDefinition[],
+                                     actionClassNameBase: string, actionTypeNameBase: string) {
+
+  console.log(name, dashedName, simpleName, responseDef, methods, methods[0].responses, schemaObjectDefinitions);
+
+  let content = '';
+  content = getReducerImports(content);
+  content = getStateInteface(content, actionClassNameBase);
+  content = getInitialState(content, actionClassNameBase);
+  content = getReducerDefinition(content, actionTypeNameBase, actionClassNameBase);
+
+  const reducersFileName = path.join(config.dest, conf.formDir + `/${dashedName}/states`, `reducers.ts`);
+  writeFile(reducersFileName, content, config.header);
+}
+
+export function getReducerImports(content: string) {
+  content += `import * as actions from './actions';\n`;
+  content += `\n`;
+  return content;
+}
+
+export function getStateInteface(content: string, actionClassNameBase: string) {
```

Comment:
It would be handy to get help with defining the reducer state interface and initialState based on the corresponding response object.